index.d.ts

```diff
@@ -106,7 +106,7 @@ declare module 'focus-trap' {
      * By default, focus trap on deactivation will return to the element
      * that was focused before activation.
      */
-    setReturnFocus?: FocusTarget;
+    setReturnFocus?: FocusTargetOrFalse;
```

Comment:
Now that we're passing a parameter to the option if it's a function, I think we need to adjust the typings here, otherwise, someone using TypeScript will have a hard time using the parameter.

Now I'm not a TypeScript expert, so while there may be a fancy way to express this using some type of template parameter or type tooling function, it's probably easiest to just not use `FocusTargetOrFalse` and write it all out:

```TypeScript
setReturnFocus?: HTMLElement | SVGElement | string | false | ((nodeFocusedBeforeActivation: HTMLElement | SVGElement) => HTMLElement | SVGElement | false);
```